### DISCUSSION TITLE
[Enhancement Idea] Media subheaders

### DISCUSSION THREAD

**Discussion by sircharlo** [sircharlo/meeting-media-manager#1376] [Ideas] [answered]
Would it be helpful to have media subheaders?

---

MW mockup:

![image](https://user-images.githubusercontent.com/1275666/217091175-97c4b85e-ab1b-4dbb-98dc-8dea671154dd.png)

The localized header values for the MW meeting could be easily extracted from the JWPUB file; everything after and including the second song is in the Living as Christians section. 

As to automatically separating the Treasures and Apply sections, that might be a bit tougher.

---

WE mockup:

![image](https://user-images.githubusercontent.com/1275666/217092223-94b2303b-bf4f-4845-ab07-6b53db55641d.png)

The localized header value for the Watchtower article could easily extracted from the JWPUB file.

---

It's just a thought.

**Comment by Chipperchap** [comment]
One "YES" vote from me... great idea! 😁

As far as separating the Treasures / Apply - most weeks not necessary as there is no student media to display (at least we don't use any).

**Comment by e-Feitosa** [comment]
I really liked the idea

You can use the following reference:

01-00 (01 - TREASURES FROM GOD'S WORD)

02-00 (02 - APPLY YOURSELF TO THE FIELD MINISTRY)

03-00 (03 - LIVING AS CHRISTIANS)

In short:
The number will correspond to each section of the meeting

**Comment by mtdvlpr** [comment]
I like it! I think it can help in scanning the media files to find the media you're looking for faster.

**Reply by mtdvlpr** [reply]
From what I found we can extract the titles from FeatureTitle column in the Document table:
- Treasures has class 21
- Living as Christians has class 10
- Apply yourself has class 94

**Reply by e-Feitosa** [reply]
I am available to test the versions (Alpha / Beta) of the M3.

**Reply by mtdvlpr** [reply]
@e-Feitosa, thanks! [Here](https://drive.google.com/drive/folders/1_2CzKieYGaK3LcYrEwzNQgfTpNReOSx9?usp=sharing) you go :)

**Comment by mtdvlpr** [comment, marked as answer]
This feature has been implemented and will be in the next release :)

**Reply by e-Feitosa** [reply]
In testing, it had an error the week of March 6th, there is a video in the do your best in ministry section. And the video was not downloaded, the publication was not found (error) and the other media downloaded normally, only the video did not download.

⬇️
![image](https://user-images.githubusercontent.com/90984283/217926559-70a8e0ba-d460-4d61-9d66-33441e45c46e.png)

**Reply by mtdvlpr** [reply]
@e-Feitosa that's the memorial first talk video. It's also not available in JW Library yet, so not an issue at our side.

**Reply by e-Feitosa** [reply]
Great!
It was very good with this adjustment in the layout. Congratulations on the job!
